toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -97,8 +97,8 @@ elfutils-libelf-lang-0.185-1.cm2.x86_64.rpm
 expat-2.4.1-1.cm2.x86_64.rpm
 expat-devel-2.4.1-1.cm2.x86_64.rpm
 expat-libs-2.4.1-1.cm2.x86_64.rpm
-libpipeline-1.5.5-1.cm2.x86_64.rpm
-libpipeline-devel-1.5.5-1.cm2.x86_64.rpm
+libpipeline-1.5.0-5.cm2.x86_64.rpm
```

Comment:
Is there any reason why libpipeline version should change with this PR ? Weird thing being that you go to a lower version (could it be a merge conflict ?)